brave/browser-laptop
# Adding single source support for publisher verification check

Fixes: #14691

## Submitter Checklist:


## Test Plan:
You must be using the `bat-client` off of this branch for testing: https://github.com/brave-intl/bat-client/pull/87

1. Run Brave with a clean profile using LEDGER_VERBOSE=true with this branch.
2. Enable Payments and wait 15 seconds or so for terminal log confirming the publishers file has been written. Check in the user data directory for `publisher-data.json`
3. Stop and restart Brave. Wait a bit and confirm via terminal that the publishers file hasn't been rewritten (should occur every 48 hours)
4. Visit a known verified publisher, wait the minimum amount of time before logging a visit.
5. Confirm via the publishers toggle/ledger table that the publisher shows as verified.
6. Perform steps 4 and 5 for a non-verified publisher.

## Reviewer Checklist:


Tests
